### PR TITLE
[EOC-112] Add stylelint

### PR DESCRIPTION
**Pull request by adamklepacz**
There is an `npm run stylelint` to lint all the files in project.
There is a webpack stylelint plugin that will not allow you to build, compile when there are stylint errors.
And there is a husky hook, which will not allow you to commit when staged style files have errors.

To use stylelint you need to install it globally. npm i stylelint -g

P.S All styles bugs are removed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,7 +99,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
+      "pre-commit": " lint-staged",
```
Comment (alemikolo) [inline package.json]: remove space

File: package.json [hunk 2]
```diff
@@ -86,6 +87,10 @@
     "redux-mock-store": "1.5.3",
     "sass-loader": "7.1.0",
     "style-loader": "0.23.0",
+    "stylelint": "9.10.1",
+    "stylelint-config-recommended": "2.1.0",
+    "stylelint-scss": "3.5.4",
+    "stylelint-webpack-plugin": "0.10.5",
```
Comment (marekrozmus) [inline package.json]: please check if this is needed

File: README.md [hunk 3]
```diff
@@ -85,6 +85,10 @@ npm start
 To run project within Docker containers simply clone this repo, install Docker on your machine and the in the root foolder of this app run
 `docker-compose up`. There will be three containers running, one for the client, second for the server, third for the database. The app will be available under `localhost:3000` .
 
+### Stylelint
+
+To use stylelint you need to install it globally. `npm install stylelint -g`
```
Comment (marekrozmus) [inline README.md]: please double check if this is needed

**Comment by adamklepacz** [comment]
Zrzut ekranu 2019-04-04 o 11 44 17
Zrzut ekranu 2019-04-04 o 11 44 47

**Pull request merged**